ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -237,9 +237,15 @@ message DamlPackageUploadRejectionEntry {
     string details = 1;
   }
 
+  // Duplicate submission with same (participant_id, submission_id) as a previous one
```

Comment:
repeating these is really starting to rub me the wrong way. I'd consider merging at least the ones that are clearly the exact same error.

Reply:
Extracted all errors to a common section at the bottom of the file